### DISCUSSION TITLE
How do I disable the hidden lines in the files diff ?

### DISCUSSION THREAD

**Discussion by aashishsingla567** [gitkraken/vscode-gitlens#3211] [Q&A]
I want to search a for particular keyword throughout the file history. 

<img width="366" alt="image" src="https://github.com/gitkraken/vscode-gitlens/assets/70276198/e7fd0e26-adb8-4326-b0a1-0990eca67b9c">
The hidden lines feature is very annoying . I want to temporarily disable this. Is it possible to do so ?

### Suggestions : 
1. Expand the hidden lines when the find navigates to the searched keyword.
2. Provide an option in command palette to toggle this feature easily.

**Comment by eamodio** [comment]
There is a "Toggle Collapse Unchanged Regions" button on diffs:

![image](https://github.com/gitkraken/vscode-gitlens/assets/641685/a62b8dc5-0a50-4abf-be80-8531b88b96d3)

This is not a GitLens feature, and is just a feature of diffs in VS Code.
